src/test/java/DeadlineTest.java

```diff
@@ -0,0 +1,30 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import org.junit.jupiter.api.Test;
+
+import task.Deadline;
+import task.Task;
+
+public class DeadlineTest {
+
+    @Test
+    public void deadlineConstructTest() {
```

Comment:
An alternative way of naming these tests is `featureUnderTest_testScenario_expectedBehavior()`, although this should not be compulsory.